dts/arm/st/f3/stm32f3.dtsi

```diff
@@ -40,6 +40,9 @@
 			flash0: flash@8000000 {
 				compatible = "soc-nv-flash";
 				label = "FLASH_STM32";
+
+				write-block-size = <16>;
```

Comment:
According to the datasheet, the flash should be written half-word by half-word. The write-block-size value is expressed in bytes and not bits, so it should be 2 here.

Reply:
Fixed, txs!